lib/matplotlib/transforms.py

```diff
@@ -1191,9 +1191,9 @@ class Transform(TransformNode):
     - :attr:`input_dims`
     - :attr:`output_dims`
     - :meth:`transform`
-    - :attr:`is_separable`
-    - :attr:`has_inverse`
-    - :meth:`inverted` (if :attr:`has_inverse` is True)
+    - :attr:`is_separable` (defaults to True for 1d -> 1d transforms)
```

Comment:
> Subclasses of this class should override the following members (at minimum):

Is not correct anymore for `is_separable` and `has_inverse`. Either rephrase or put those two in a separate block.

Reply:
good point, done